fastlane/Fastfile

```diff
@@ -268,7 +270,7 @@ def check_no_git_tag_exists(version_number)
   end
 end
 
-def update_sample_podfile_lock_with_retry(pod_name)
+def update_sample_podfile_lock_with_retry(pod_name, sample_path)
```

Comment:
`sample_path` is already defined, though, right? why does passing it here make a difference?

Reply:
had to ask @joshdholtz (thanks!) about this hehe

`sample_path` is a local variable, lanes are blocks which have access to local variables but methods cannot access local variables outside of their scope. This method could be a private lane which would make it a block, and therefore it would have access to the local variable. The alternative is to pass it as a parameter like I did here

Reply:
ohhhh that makes sense, thanks